### PR TITLE
sel activities by skill category page created

### PR DESCRIPTION
**Pull request by delacruzralph**

### CODE REVIEW

File: src/components/SelCard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import LinearProgressWithLabel from '@mui/material/LinearProgress'
+
+const SelCard = ({ title, description, progress }) => {
```
Comment (jaanatabalon) [inline src/components/SelCard.js:7]: should we pass color in as a parameter?

**Review by jaanatabalon — COMMENTED** [review]
pass color in as a parameter?

File: src/components/SelCard.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
```
Comment (liu-caro) [inline src/components/SelCard.js:3]: delete unused import

File: src/components/SelCard.js (hunk 1 above)
Comment (liu-caro) [reply src/components/SelCard.js:7]: agree with Jaana - can add color as a parameter and pass style into the Card component like `        <Card sx={{ maxWidth: 500 }} style={{ backgroundColor: color}}>
`

**Review by liu-caro — CHANGES_REQUESTED** [review]

File: src/screens/ActivitiesPage.js [hunk 3]
```diff
@@ -1,8 +1,56 @@
...
+      <Grid item xs={12}>
+        <Typography variant="h2" align='center'>SEL Activities by Skill Category</Typography>
+      </Grid>
+
+      <Grid item m={1}>
+        <SelCard
+          title={'Responsible Decision Making'}
+          description={'Understanding how our decisions can impact others and ourselves'} 
+          progress={72} />
+      </Grid>
```
Comment (liu-caro) [inline src/screens/ActivitiesPage.js:23]: Make sure to link these to their respective pages as well - check how I did it in Card component.

**Pull request merged**